File: c++/001.cpp
// Multiples of 3 or 5

#include <iostream>

using namespace std;

int main()
{
    int i = 0, sum = 0;

    do
    {
        if (i % 3 == 0 || i % 5 == 0)
            sum += i;

        i += 1;
    } while (i < 1000);
    
    cout << sum;
}

File: rust/001.rs
// Multiples of 3 or 5

fn main() {
    let mut count = 0u32;
    let mut _sum = 0u32;

    loop {
        if count == 999 {
            break;
        }

        count += 1;

        if count % 3 == 0 || count % 5 == 0 {
            _sum += count;
        }
    }

    println!("{}", _sum);
}
